pkg/test/framework/components/istio/flags.go

```diff
@@ -65,4 +65,12 @@ func init() {
 		settingsFromCommandline.EgressGatewayIstioLabel,
 		`Specifies the istio label of the egressgateway to search for when running tests in a preinstalled istio installation.
 		Should only be set when istio.test.kube.deploy=false`)
+	flag.StringVar(&settingsFromCommandline.UseSharedMeshConfig, "istio.test.kube.useSharedMeshConfig",
```

Comment:
nit: no need for the flag? `sharedMeshConfigName` being non-empty implies to use it?